docker/ubuntu-focal-loadtest/startup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+set -e
+
+function cleanup {
+if [ -n "$log_file" ] && [ -f "$log_file" ] ; then
+    gsutil cp "$log_file" "gs://$GCE_BUCKET_NAME"
+    gcloud compute instances delete -q "$INSTANCE_ID" --zone "$GCE_ZONE"
+fi
+}
+
+trap cleanup EXIT
+
+if [ -z "$INSTANCE_ID" ]
+then
+      echo "\$INSTANCE_ID env var should be set!"
+      exit
```

Comment:
Sounds this should exit with an error code. And perhaps is not "should" but "must"

Reply:
Fixed,